editor.planx.uk/src/@planx/components/TextInput/Public.test.tsx

```diff
@@ -8,6 +8,9 @@ import { axe } from "vitest-axe";
 import { ERROR_MESSAGE } from "../shared/constants";
 import { TextInputType } from "./model";
 import TextInput from "./Public";
+
+const twentyFiveCharacterTest = "25 characters for me.....";
```

Comment:
Felt easier cause I could use  a `.repeat()` to trigger the error on the textarea